modules/selenium/src/test/java/org/testcontainers/junit/LinkedContainerTest.java

```diff
@@ -1,69 +0,0 @@
-package org.testcontainers.junit;
-
-import lombok.Cleanup;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.openqa.selenium.By;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.openqa.selenium.remote.RemoteWebDriver;
-import org.testcontainers.SeleniumTestImages;
-import org.testcontainers.containers.BrowserWebDriverContainer;
-import org.testcontainers.containers.Network;
-import org.testcontainers.containers.NginxContainer;
-
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.PrintStream;
-
-import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
-
-
-/**
- * @author richardnorth
- */
-public class LinkedContainerTest {
```

Comment:
Given that all our tests will now use 'linked' (networked) containers, there's little point in having an explicit test for it that works another way.